tests/containers/podman_pods.pm

```diff
@@ -28,13 +28,12 @@ sub run {
 
     record_info('Test', 'Confirm pods are running');
     record_info('pod ps', script_output('podman pod ps'));
-    assert_script_run('podman pod ps | grep "Running" | wc -l | grep -q 3');
+    assert_script_run('podman pod ps | grep "Running" | wc -l | grep -q 1');
```

Comment:
Just curious how this would come out for tests that are run against podman < 4.4.0? Should this be conditional?

Reply:
It would if the replica count in the yaml was still 3, but because I wanted to avoid branches here, I changed it to 1.

Reply:
You call it clever, I call it lazy ;-)